### PR TITLE
Fix ActivityButton icon aria-label issue

### LINKED ISSUE #98: (ActivityButton): icon name is displayed in title/aria-label
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-25920 & https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-25924

### PR DESCRIPTION
**Pull request by pauljeter**
fixes #98 & https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-25920 & https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-25924

### CODE REVIEW

File: src/lib/Button/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class Button extends React.Component {
     const { focusIndex, focusOnLoad } = this.context;
     (!ariaLabel && !ariaLabelledBy)
       &&
-      console.warn('Accessibility could be improved with ariaLabel');
+      console.warn('[@collab-ui/react] Button: Accessibility could be improved with ariaLabel');
```
Comment (bfbiggs) [inline src/lib/Button/index.js]: Do you want to add this prefix to the other consoles in this component like you did in Icon?

File: src/lib/Icon/index.js [hunk 2]
```diff
@@ -125,28 +131,45 @@ class Icon extends React.PureComponent {
...
         <svg
           className={
             `cui-icon` +
             `${(className && ` ${className}`) || ''}`
           }
+          name={name}
           width={getSize()}
           height={getSize()}
-          aria-labelledby={isAria ? getAria() : undefined}
+          aria-labelledby={
```
Comment (bfbiggs) [inline src/lib/Icon/index.js]: Suggest moving this logic to your getAriaLabelledBy function.

File: src/lib/Icon/index.js [hunk 3]
```diff
@@ -125,28 +131,45 @@ class Icon extends React.PureComponent {
...
+            isAria ? // TODO(pajeter): remove with next major release
+              !onClick ?
```
Comment (bfbiggs) [inline src/lib/Icon/index.js]: Ternaries at the beginning of the phrase

File: src/lib/Icon/index.js [hunk 4]
```diff
@@ -125,28 +131,45 @@ class Icon extends React.PureComponent {
         : consoleHandler('name-error', iconName);
     };
 
-    const getAria = () => (
-      description ? `${titleId} ${descId}` : `${titleId}`
-    );
+    const getAriaLabelledBy = () => {
+      if (!ariaLabel) {
+        if (title && description) return (`${titleId} ${descId}`);
```
Comment (bfbiggs) [inline src/lib/Icon/index.js]: Wouldn't you want a dash here?

File: src/lib/Button/index.js (hunk 1 above)
Comment (pauljeter) [reply src/lib/Button/index.js]: I didn't mean to leave this here. I was trying to update all of the console warnings in another PR

File: src/lib/Icon/index.js (hunk 4 above)
Comment (pauljeter) [reply src/lib/Icon/index.js]: no, these are not the title and description, but the id's of the `<title>` and `<desc>` elements for the aria-labelledby

**Pull request merged**